src/main/java/net/shipilev/Disruptor.java

```diff
@@ -24,17 +24,17 @@
 import java.util.concurrent.TimeUnit;
 
 @State(Scope.Benchmark)
-@Warmup(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Fork(5)
+@Warmup(iterations = 3, time = 1, timeUnit = TimeUnit.MILLISECONDS)
```

Comment:
Just found that `@BenchmarkMode(Mode.SingleShotTime)` do what I tried to archive by `time = 1, timeUnit = TimeUnit.SECONDS` and with better reporting by percentile histogram.

Will you accept this PR with following annotations:
```
@State(Scope.Benchmark)
@Warmup(iterations = 3)
@Measurement(iterations = 10)
@Fork(1)
@BenchmarkMode(Mode.SingleShotTime)
@OutputTimeUnit(TimeUnit.MILLISECONDS)
```
?

Main goal of switching from `AverageTime` to `SingleShotTime` mode is not hiding of per iteration results (by averaging). Minimal value does matter, especially when it occurs in first iteration.